tests/unit/via/get_url_test.py

```diff
@@ -80,21 +69,12 @@ def test_it_raises_BadURL_for_invalid_urls(self, bad_url):
         with pytest.raises(BadURL):
             get_url_details(bad_url)
 
-    @pytest.mark.parametrize(
-        "request_exception,expected_exception",
-        (
-            (MissingSchema, BadURL),
-            (ProxyError, UpstreamServiceError),
-            (SSLError, UpstreamServiceError),
-            (UnrewindableBodyError, UnhandledException),
-        ),
-    )
-    def test_it_catches_requests_exceptions(
-        self, requests, request_exception, expected_exception
-    ):
-        requests.get.side_effect = request_exception("Oh noe")
+    def test_it_catches_requests_exceptions(self, requests):
+        # We don't need to go crazy here, just enough to prove we have the
+        # decorator applied
```

Comment:
```suggestion
```

I think it's unclear what _go crazy_ or _the decorator_ refer to. Especially if the code changes in future so that the function has more than one decorator or so that `@handle_errors` is no longer a decorator, this comment is unlikely to get updated if that happens

Reply:
I'll reword this